src/validate_historic_quantile_data.R

```diff
@@ -35,7 +36,7 @@ validate_historic_quantile_data <- function(quantile_data, n_hrus = 109951, n_da
   # Max of total storage in all of historic total storage data for all HRUs is ~8,000 mm
   # Considering 300 mm is about 1 ft of water (diff between historic max and assumed max is ~ 6.5 ft of water)
   # An absolute max for order of magnitude check of 10,000 mm seems appropriate
-  just_quantile_data <- as.matrix(quantile_data[,real_cols])
+  just_quantile_data <- as.matrix(quantile_data[,real_cols, with=FALSE])
```

Comment:
Huh, I wonder if `with = FALSE` is an addition needed in a newer version of data.table. Just curious why it is needed for my own learning.

Reply:
yeah, that is actually something you added way back when (in September, hahahah) we tried to do this the first time 👍

Reply:
bahahahahaha oh man 🤦

Reply:
i know. i know.